buildscripts/Dockerfile

```diff
@@ -0,0 +1,40 @@
+
+FROM ubuntu:14.04
+
+MAINTAINER Minio Community
+
+ENV GOLANG_TARBALL go1.4.2.linux-amd64.tar.gz
+
+ENV GOROOT /usr/local/go/
+ENV GOPATH /go-workspace 
+ENV PATH ${GOROOT}/bin:${GOPATH}/bin/:$PATH
+
+RUN apt-get update -y && apt-get install -y -q \
+		curl \
+		build-essential \
+		ca-certificates \
+		git yasm \
+		python-pip
+
+RUN curl -O -s https://storage.googleapis.com/golang/${GOLANG_TARBALL} && \
+		tar -xzf ${GOLANG_TARBALL} -C ${GOROOT%*go*} && \
+		rm ${GOLANG_TARBALL} 
+
+RUN pip install mkdocs
+
+RUN mkdir -p ${GOPATH}/src/github.com/minio-io/
```

Comment:
yes, that way would be clearly better. I guess I have to move Dockerfile because Docker doesn't like files outside its context